api/py/test/sample/production/joins/sample_team/sample_join.group_by_of_group_by

```diff
@@ -39,7 +39,7 @@
           "production": 0,
           "customJson": "{\"lag\": 0, \"groupby_tags\": null, \"column_tags\": {}}",
           "dependencies": [
-            "{\"name\": \"wait_for_sample_namespace.sample_team_sample_group_by_require_backfill_ds\", \"spec\": \"sample_namespace.sample_team_sample_group_by_require_backfill/ds={{ ds }}\", \"start\": \"2021-04-09\", \"end\": null}"
+            "{\"name\": \"wait_for_sample_namespace.sample_team_sample_group_by_group_by_require_backfill_ds\", \"spec\": \"sample_namespace.sample_team_sample_group_by_group_by_require_backfill/ds={{ ds }}\", \"start\": \"2021-04-09\", \"end\": null}"
```

Comment:
any reason this changed? Did the naming change?

Reply:
@rohitgirme This is weird since I didn't update the python file of this group_by. I guess it is just re-generated by the unit test command. I checked the history, this groupBy's python config was added in 2023 but the json config was added very recently by you. Did you make some change to add the json config?

Reply:
No. I materialized it for use with the dependency tracker <> compile tests.